install/kubernetes/cilium/values.yaml.tmpl

```
@@ -1340,6 +1340,9 @@ hubble:
             excludeFilters: []
   # -- Unix domain socket path to listen to when Hubble is enabled.
   socketPath: /var/run/cilium/hubble.sock
+  # -- Enable L3/L4 network policy enrichment of Hubble Flows (true by default).
```

I wonder if we should clarify in the docstring here a bit more clearly what this is (since "enrichment" is a very generic term). I even document what fields we're talking about here to make it clear.

```suggestion
  # -- (bool) Enable network policy enrichment of Hubble flows, i.e. populating `egress_allowed_by`, `ingress_denied_by` fields with policy information.
  # @default -- `true`
```

I would also suggest to use the Helm schema commands that we use rather than documenting default values in text.

---

I left the property as an object and made the updates now.